org.jacoco.core/src/org/jacoco/core/internal/flow/MethodProbesVisitor.java

```diff
@@ -147,4 +148,8 @@ public void visitLookupSwitchInsnWithProbes(final Label dflt,
 			final int[] keys, final Label[] labels, final IFrame frame) {
 	}
 
+	public void accept(final MethodNode methodNode,
```

Comment:
Whooo, here we start mixing the ASM event and tree API. I consider this as a temporary hack until we fully move to the tree API ;-)

Nevertheless we should clearly document the method, especially the point in time when it is called.